examples/Cargo.toml

```diff
@@ -25,7 +25,8 @@ term = { git = "https://github.com/Stebalien/term", rev = "11e754980706a53dda998
 
 # Patch dependencies on oak crates so that they refer to the versions within this same repository.
 oak = { path = "../sdk/rust/oak" }
+oak_abi = { path = "../oak/server/rust/oak_abi" }
 oak_derive = { path = "../sdk/rust/oak_derive" }
 oak_log = { path = "../sdk/rust/oak_log" }
-oak_runtime = { path = "../sdk/rust/oak_runtime" }
+oak_runtime = { path = "../oak/server/rust/oak_runtime" }
```

Comment:
Perhaps not in this PR, but should we collapse the `rust` subdirectories at some point ? e.g. `oak/server/rust` -> `oak/rust` ? This might depend on if we choose to go "full rust"

Reply:
I was thinking the same (that it would happen, but only when we go full Rust)